### PR TITLE
Updated Bibtex references in plot_receptive_field.py

### LINKED ISSUE #8936: use bibtex in plot_receptive_field.py
convert references in `tutorials/machine-learning/plot_receptive_field.py` to use footcite / footbibliography

### PR DESCRIPTION
**Pull request by ramkpari**
#### Reference issue
Fixes #8936 .

#### What does this implement/fix?
Updated references in tutorials/machine-learning/plot_receptive_field.py to Bibtex format. Also moved the references section to the bottom of the document to match styling with  plot_sensors_decoding.py and to use `footcitations` / `footbibliography`.

### CODE REVIEW

File: tutorials/machine-learning/plot_receptive_field.py [hunk 1]
```diff
@@ -364,3 +341,8 @@
 plt.setp([iax.get_xticklabels() for iax in [ax1, ax2, ax3]], rotation=45)
 plt.autoscale(tight=True)
 mne.viz.tight_layout()
+
+###############################################################################
+# References
+# ==========
+# .. footbibliography::
```
Comment (drammock) [inline tutorials/machine-learning/plot_receptive_field.py:348]: missing newline at end of file.  Our style checker will complain.  Other than that the changes look reasonable.

Comment (ramkpari) [reply tutorials/machine-learning/plot_receptive_field.py]: Latest commit ( 287cf81 ) should fix that !

**Review by drammock — APPROVED** [review]

**Review by agramfort — APPROVED** [review]
+1 for MRG when green

**Comment by larsoner** [comment]
Thanks @ramkpari !

**Pull request merged**